Amazing print

qortex opened the pull request:
Fixes #3 .

Awesome Print / Amazing Print have been removed from the gemspec altogether (as it is the case in `Ougai`). Since the package is conditionnally loaded and `LoadError` is watched for, it seems more coherent.

qortex committed:
Replace Awesome Print with Amazing print

qortex committed:
Run bundler install

qortex commented:
Ah looks like new versions of ougai needs more recent Ruby version.

Is it an issue for you to merge or you're okay with bumping up the Ruby version?

Al-un commented:
@qortex 
Thanks a lot for the PR! I haven't done any Ruby for almost a year but I lost a bit track of everything. 

As for failed Travis job, yes, it seems the bumping Ougai requires bumping the Ruby version. I am OK for bumping Ruby version (currently using 2.6.x in my current machine anyway). Can the ruby version bump be added in this PR?

qortex committed:
Bump minimum ruby version for ougai compatibility & add amazing_print as dev dependency for tests to pass

qortex commented:
Thanks for your feedback!

Added the `>= 2.4.0` requirement on ruby, removed the `2.3.0` and bumped test versions for ruby.

Also added the `amazing_print` as dev dependency for tests to pass.

Looks ok now :)

Al-un commented:
Wow the tests I wrote did not break :astonished: 

Thanks a lot @qortex ! LGTM so merging this PR

qortex commented:
Great, cool :)
Could you release a new version as a result? Thanks a bunch!

qortex commented:
(Without a release, bundler doesn't get it with `update`, so it is cumbersome when using cascading gems)
